community/modules/scripts/htcondor-install/files/autoscaler.py

```diff
@@ -226,29 +232,21 @@ def scale(self):
             print("Information about the compute instance template")
             pprint(instanceTemplateInfo)
 
-        # Calculate number instances to satisfy current job queue CPU requests
-        if queue > 0:
-            self.size = int(math.ceil(float(queue) / float(self.cores_per_node)))
```

Comment:
nit. Are we sill supporting  Python2 ? If no, `float` and `int` casts can be removed.